Source/Core/EllipsoidTangentPlane.js

```diff
@@ -230,5 +314,56 @@ define([
         return result;
     };
 
+    var scratchOffset = new Cartesian3();
+    var scratchScale = new Cartesian3();
+    /**
+     * Computes an OrientedBoundingBox given extents in the east-north-up space of the tangent plane.
+     *
+     * @param {Number} minX Minimum x extent in tangent plane space.
```

Comment:
With the exception of local identifiers (and some rare cases like WGS84), do not use abbreviations.

Spell out minimum and maximum here.